### PR TITLE
Add missing documentation

### PR DESCRIPTION
**Pull request by corinnekrych**
@jcesarmobile mind a quick review?

### CODE REVIEW

**Comment by jcesarmobile** [comment]
👀

File: FeedHenry/FH.swift [hunk 1]
```diff
@@ -204,36 +245,84 @@ open class FH: NSObject {
         })
     }
     
+    /**
+     Utility method that registers to AeroGear Unified Push Server and configure an alias.
+     - parameter alias: is a string to alias the device where to send push notification.
```
Comment (jcesarmobile) [inline FeedHenry/FH.swift]: notifications

File: FeedHenry/FH.swift [hunk 2]
```diff
@@ -204,36 +245,84 @@ open class FH: NSObject {
...
+     - parameter success: closure to run upon success of the push registration,
```
Comment (jcesarmobile) [inline FeedHenry/FH.swift]: registration.

File: FeedHenry/OAuth/OAuthViewController.swift [hunk 3]
```diff
@@ -30,10 +34,12 @@ open class OAuthViewController: UIViewController, UIWebViewDelegate {
         self.view.addSubview(webView)
     }
     
+    /// Overrode to deal with error.
```
Comment (jcesarmobile) [inline FeedHenry/OAuth/OAuthViewController.swift]: Override

**Comment by corinnekrych** [comment]
@jcesarmobile ok to merge?

File: FeedHenry/FH.swift [hunk 4]
```diff
@@ -193,47 +235,97 @@ open class FH: NSObject {
...
+                                response.error = err
+                                error(response)
         })
     }
     
+    /**
+     Utility method that registers to AeroGear Unified Push Server and configure an alias.
+     
+     - parameter alias: is a string to alias the device where to send push notifications.
+     - parameter success: closure to run upon success of the push registration,
```
Comment (jcesarmobile) [inline FeedHenry/FH.swift]: registration, -> registration.

File: FeedHenry/FH.swift [hunk 5]
```diff
@@ -175,13 +188,42 @@ open class FH: NSObject {
             }
         }
     }
-    
+    /**
+     Register for remote notification in AppDelegate's lifecycle method.
```
Comment (jcesarmobile) [inline FeedHenry/FH.swift]: notifications

**Pull request merged**